doc/en/api/1.0.0/featuretypes.yaml

```diff
@@ -631,7 +632,7 @@ paths:
           type: string
         - name: recalculate
           in: query
-          description: Specifies whether to recalculate any bounding boxes for a feature type. Some properties of feature types are automatically recalculated when necessary. In particular, the native bounding box is recalculated when the projection or projection policy are changed, and the lat/lon bounding box is recalculated when the native bounding box is recalculated, or when a new native bounding box is explicitly provided in the request. (The native and lat/lon bounding boxes are not automatically recalculated when they are explicitly included in the request.) In addition, the client may explicitly request a fixed set of fields to calculate, by including a comma-separated list of their names in the recalculate parameter.  The empty parameter 'recalculate=' is useful avoid slow recalculation when operating against large datasets as 'recalculate=' avoids calculating any fields, regardless of any changes to projection, projection policy, etc. The nativebbox parameter 'recalculate=nativebbox' is used recalculates the native bounding box, while avoiding recalculating the lat/lon bounding box. Recalculate parameters can be used in together - 'recalculate=nativebbox,latlonbbox' can be used after a bulk import to recalculate both the native bounding box and the lat/lon bounding box.
```

Comment:
That is a hard one to take, losing the customized attributes; any hesitation about this being the right thing to do?

Reply:
No hesitation so far... let me explain.

If I just want to reset the knowledge that GeoServer has about the original attributes, I'll just use reset, and pray the 
However, if those changed, there is a significant chance the customization will not work anymore (it might referer to attributes that are no longer there). At that point there are two choices:
* Reset the customization and start fresh, maybe to customize later (I'm imagining a UI workflow to do that)
* Somehow rebuild a sensible customization right away, doing a PUT against the FeatureType resource that already contains the new list of attribute infos.